src/main/java/serhat/workshop_3/Task2.java

```diff
@@ -0,0 +1,56 @@
+package serhat.workshop_3;
+
+/**
+ * Created by serhatgemici on 07/02/2017.
+ */
+public class Task2 {
+
+    private static void printTheLongestWord(String sentence) {
+        sentence.trim();
+        String initialLongestWord = "";
+        char partial;
+        String realLongestWord = "";
+
+        for (int i = 0; i < sentence.length(); i++) {
+
+            if ((sentence.charAt(i) != ' ') && (sentence.charAt(i) != '!') && (sentence.charAt(i) != '.') && (sentence.charAt(i) != ',') && (sentence.charAt(i) != '\'')) {
+
+                partial = sentence.charAt(i);
+                initialLongestWord = initialLongestWord + partial;
+
+            } else if (sentence.charAt(i) == ' ' || sentence.charAt(i) == '!') {
+
+                if ((initialLongestWord.length() > realLongestWord.length()) || (initialLongestWord.length() == realLongestWord.length())) {
+
+                    realLongestWord = initialLongestWord;
+
+                }
+
+                initialLongestWord = "";
+
+            }
+        }
+
+        System.out.println("In the sentence:");
+        System.out.println("\"" + sentence + "\"");
+        if(sentence.length()==0){
+
+            System.out.println("The longest cannot be calculated, because there is no sentence");
+            System.out.println("");
```

Comment:
what will you print by that? if there is a new line than \n could be more useful on my opinion